include/fdreadoutlibs/FDReadoutTypes.hpp

```diff
@@ -431,67 +431,64 @@ struct RAW_WIB_TRIGGERPRIMITIVE_STRUCT
   RAW_WIB_TRIGGERPRIMITIVE_STRUCT()
   {
     m_raw_tp_frame_chunksize = 0;
+    m_first_timestamp = 0;
   }
 
-  using FrameType = dunedaq::detdataformats::wib::RawWIBTp;
+  using FrameType = RAW_WIB_TRIGGERPRIMITIVE_STRUCT;
```

Comment:
Why are we using the same aggregator type for FrameType in its aggregator struct/type?

Reply:
Hi @roland-sipos ,

Thanks for pointing this out. It is a good question indeed. I think it is a matter of choice and I made this 
choice now. It is currently similar to SW_WIB_TRIGGERPRIMITIVE_STRUCT.
I would suggest to keep this choice for now, however if you have a suggestion of other aggregator type, please let us know. The choice depends on how RAW_WIB_TRIGGERPRIMITVE_STRUCT is used, and in this PR there is not other critical functionality that relies on it. Please let us know what you think and how to proceed with this PR.